docs/setup/kubeedge_configure.md

```diff
@@ -0,0 +1,369 @@
+# KubeEdge Configuration
+
+KubeEdge requires configuration on both [Cloud side (KubeEdge Master Node)](#configuration-cloud-side-kubeedge-master-node) and [Edge side (KubeEdge Worker Node)](#configuration-edge-side-kubeedge-worker-node)
```

Comment:
I didn't like the name Master Node. Can we delete Node.

Node indicates, that the cloud side has to be executed on a dedicated node. But it is possible to execute cloud side as pod in an existing kubernetes cluster. (comparing to https://github.com/johscheuer/kubeedge-setup)